grpc/common/common.go

```diff
@@ -178,6 +190,53 @@ func ToCoreMetrics(mts []*Metric) []core.Metric {
 	return metrics
 }
 
+// Convert slice of common.Metric to []core.RequestedMetric
+func ToRequestedMetrics(mts []*Metric) []core.RequestedMetric {
+	metrics := make([]core.RequestedMetric, len(mts))
+	for i, mt := range mts {
+		metrics[i] = ToCoreMetric(mt)
+	}
+	return metrics
+}
+
+func RequestedToMetric(requested []core.RequestedMetric) []*Metric {
+	reqMets := []*Metric{}
```

Comment:
Same issue as mentioned before, strange in function above it's done correctly :+1: